### ISSUE TITLE
device kind or "is-a" type field

### ISSUE THREAD

**Issue by tappdarden** [vincentbernat/wiremaps#17]
I hope my "issues" is a show of interest and not too bothersome...
what about some way to know what "type" of a device something is? this could be a vendor provided model description, or just whatever OID that wiremaps is using to grab the image.

I'm making an client that is using the API. it would be nice to know if i need to download a new image or just use a previously downloaded image for another device (of the same type).

in other words, in the API you can supply wiremaps a oid and it will give you the image for that oid. i would like some way to know what the oid is for a specific device...
